testproject/requirements.txt

```diff
@@ -28,5 +28,7 @@ channels==4.0.0
 channels_redis==4.0.0
 
 django-notifications-hq @ git+https://github.com/openwisp/django-notifications.git@f4918c2409a734ec1aa9798d86c3a729b505f753
+django-push-notifications[WP,apns-async,FCM] @ git+https://github.com/jazzband/django-push-notifications.git@7074131559273edad993c6c227803c53f988d1b8
```

Comment:
I had to point to this commit because the version which fixes the default_auto_field warning isn't released yet. I've added myself to get notifications when the new version gets released.